### DISCUSSION TITLE
Is there any simple way to archive daily notes?

### DISCUSSION THREAD

**Discussion by ruipreis** [ivan-lednev/obsidian-day-planner#445] [Q&A] [answered]
I have a folder with a lot of daily notes, I’ve been taking notes for about 6 months, and it has become a real hassle to manage such a giant folder.

Initially I created a UNIX command to manually keep the N newest notes and move the remaining to an archive folder, but this is suboptimal since I lose the ability to view these notes in the planner timeline. The situation has worsen now that I had to move to Windows, which prohibits me from using the UNIX commands.

Is there any to achieve this using just the plugin?

**Comment by ivan-lednev** [comment, marked as answer]
Hello, @ruipreis, there is no such feature in the plugin, as this would be outside of its scope. As a workaround, you might try to use Git Bash, which allows you to run Bash scripts on Windows.

**Comment by jessycormier** [comment]
@ruipreis a solution for things getting out of hand with daily notes (I have notes going back as far as 2006) is to set up a few things in the core daily notes settings.

Set the **Date format** to something like this `YYYY/MM-MMMM/YYYY-MM-DD--dddd`.
Notice how it's creating folders `/`. this works perfecting and will generate named folders.

To have things more organized you can also set **New file location** to something so you're daily notes would appear in something like `Notes/2025/02-Feb/2025-02-06--Thursday.md`. This keeps everything tidy.

You can still if you no longer need anything in the older notes it's way easier to _archive_ them when their organized by year/month.

Hope that helps you.
